src/home/migrations/0027_manual_paragraph_data.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.10 on 2020-04-02 21:08
+
+from django.db import migrations
+from itertools import chain
+from utils.data_migrations import stream_field_filter_map
+
+def richtext_to_paragraph(block):
+    return {
+        'type': 'paragraph',
+        'value': {
+            'text':  block['value'],
+        }
+    }
```

Comment:
You must set a default alignment when converting to a paragrah. Otherwise it will be empty but the model requires that it has a value. Left-align is probably a good default?

Reply:
Yes, it won't affect how the pages look after migrations but you will be prompted to set them to a valid value whenever you edit the page. I suppose that it can be annoying, will fix soon. 

I think it is possible to remove the "----" option from choice blocks if you define your own choice block as a value-block wrapping a native django choice-field. IIRC I did something similar in previous PR before I found out that wagtail already supplies a choice block. Might be a future improvement to look into.